lib/util/save-async-stack.js

```diff
@@ -2,8 +2,13 @@ module.exports = function saveAsyncStack(instance, lines) {
   if (instance.client.config.asyncStackTraces) {
     // a hack to get a callstack into the client code despite this
     // node.js bug https://github.com/nodejs/node/issues/11865
```

Comment:
So from my understanding this could be removed after Node 10 support is dropped, is that correct?

Reply:
Can't be removed even after node 10, node stil has some issues with async traces